File: 2020/Python/Junior Problems/J2.py
list1 = []
for m in range(3):
    ask = int(input(''))
    list1.append(ask)
count = 0
num = list1[1]
num2 = list1[1]
for n in range(100000000000):
    count+=1
    num2=num2*list1[2]
    num+=num2
    if num>list1[0]:
        print(count)
        exit()

File: 2020/C++/Junior Problems/J2.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    int P,N,R;
    cin >> P >> N >> R;
    int num = N;
    int num2 = N;
    int c = 0;
    while (true){
        c++;
        num2 = num2*R;
        num+=num2;
        if (num>P){
            cout << c;
            break;
        }
    }
    return 0;
}
